src/app/supplies/page.tsx

```diff
@@ -45,18 +55,52 @@ const MUITable = () => {
     setFilteredRows(rows);
   };
 
-  const fetchAvailableSupplyData = async () => {
+  const fetchSupplyPointsData = async () => {
     try {
       const response = await apiClient.get("/supplies", {
-        withCredentials: true
+        headers: {
+          "Content-Type": "application/json",
+        },
+        data: {}, // Añadir cuerpo vacío para solucionar error de "Content-Type vacío"
       });
-    } catch (error) {
-      console.error("Error retrieving supplies:", error);
+
+      console.log("Response data:", response.data);
+      const supplies = transformToRows(response.data.items);
+      setFilteredRows(supplies);
+    } catch (error: any) {
+      if (error.response) {
+        console.error("Error response data:", error.response.data);
+        console.error("Error status:", error.response.status);
+        console.error("Error headers:", error.response.headers);
+      } else if (error.request) {
+        console.error("Error request:", error.request);
+      } else {
+        console.error("Error message:", error.message);
+      }
+      console.error("Error config:", error.config);
```

Comment:
Remember to remove all console logs after finishing a task.

Reply:
I have found a popular logging library for React called [Winston](https://betterstack.com/community/guides/logging/how-to-install-setup-and-use-winston-and-morgan-to-log-node-js-applications/). I think we could integrate it into our project.